frontend/src/utils/constants.ts

```diff
@@ -12,6 +12,10 @@ export const headerLinks: Link[] = [
       { text: 'Snapshots', href: '/snapshots' },
     ],
   },
+  {
+    text: 'mentorship',
```

Comment:
This should start with capital letter, as other items in the header:
```suggestion
    text: 'Mentorship',
```